cmd/lakectl/cmd/root.go

```diff
@@ -71,9 +71,10 @@ type Configuration struct {
 		Provider        struct {
 			Type   lakefsconfig.OnlyString `mapstructure:"type"`
 			AWSIAM struct {
-				TokenTTL            time.Duration     `mapstructure:"token_ttl_seconds"`
-				URLPresignTTL       time.Duration     `mapstructure:"url_presign_ttl_seconds"`
-				TokenRequestHeaders map[string]string `mapstructure:"token_request_headers"`
+				TokenTTL            time.Duration      `mapstructure:"token_ttl_seconds"`
+				URLPresignTTL       time.Duration      `mapstructure:"url_presign_ttl_seconds"`
+				RefreshInterval     time.Duration      `mapstructure:"refresh_interval"`
+				TokenRequestHeaders *map[string]string `mapstructure:"token_request_headers"`
```

Comment:
no action item here in pr, just verify with product that users who will configure this in lakectl.yaml in order to run in fluffy - and then will try older version of lakectl will get error.